src/deploy/functions/release/fabricator.ts

```diff
@@ -354,6 +368,8 @@ export class Fabricator {
       invoker = endpoint.httpsTrigger.invoker;
     } else if (backend.isTaskQueueTriggered(endpoint)) {
       invoker = endpoint.taskQueueTrigger.invoker;
+    } else if (backend.isBlockingTriggered(endpoint)) {
+      invoker = ["public"];
```

Comment:
Yeah the out-of-band rule makes sense. I'm specifically violating it in the Auth Blocking case since the `setTrigger` will fail on non-public functions. I figured that a failed deployment should be a big enough thing to "bend" this rule. Let me know if my logic makes sense here